{"uuid": "70e90842-209d-4678-be1c-addb279b8fcb", "children": ["b4c96230-8772-4814-93e3-41680e7ae84a"], "befores": [{"name": "setup_driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\prade\\PycharmProjects\\allgeo_web_app_pro\\conftest.py\", line 56, in setup_driver\n    web_driver = webdriver.Chrome(\n                 ^^^^^^^^^^^^^^^^^\n"}, "start": 1712746780611, "stop": 1712746780611}], "start": 1712746780611, "stop": 1712746780646}